samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -141,23 +147,40 @@ static void lc3_audio_timer_timeout(struct k_work *work)
 			net_buf_unref(buf);
 		} else {
 			printk("  TX LC3 l: %zu\n", tx_sdu_len);
-			pdu_cnt++;
+			sdu_cnt++;
 		}
 	}
 }
 
 static void init_lc3(void)
 {
-	/* Fill audio buffer with Sine wave */
-	fill_audio_buf_sin(audio_buf, AUDIO_LENGTH_US, AUDIO_FREQUENCY_HZ,
-			   AUDIO_SAMPLE_RATE_HZ);
+	int freq_hz = bt_codec_cfg_get_freq(&codec_preset.codec);
+	int frame_duration_us = bt_codec_cfg_get_frame_duration_us(&codec_preset.codec);
```

Comment:
We've called `bt_codec_cfg_get_frame_duration_us` before as well. Can we reduce it to a single call?

Reply:
Yes as the assumption is that we only have one stream, I can put the value into a global variable. Ideally the application should hold a structure for each stream that holds the values. If you have multiple streams you need a per-stream value.